x-pack/plugins/grokdebugger/public/index.js

```diff
@@ -7,6 +7,6 @@
 
 import { GrokDebuggerUIPlugin } from './plugin';
 
-export function plugin(initializerContext) {
-  return new GrokDebuggerUIPlugin(initializerContext);
+export function plugin() {
+  return new GrokDebuggerUIPlugin();
```

Comment:
Did you intentionally remove the `initializerContext` arg?

Reply:
I got this one and PainlessLab confused I knew I had to add it to one of them. will update.